test/test_distributed.py

```diff
@@ -132,6 +132,26 @@ def opt_fn(inputs):
         self.assertTrue(same(correct_outputs, opt_outputs))
         self.assertEqual(check_splits_compiler.compiler_called, 3)
 
+    @pytest.mark.skipif(
```

Comment:
btw this skipif was intended as a transient thing, but maybe we should remove it now that the ddp change on pytorch side has landed and doesn't seem likely to get reverted.  not urgent.